Core/Src/main_task.c

```diff
@@ -560,19 +674,22 @@ static void run_upshift_sm(void)
 		safe_spark_cut(false);
```

Comment:
The comment at the start of FINSH_SHIFT is no longer accurate, modify it to explain what is actually happening now